### ISSUE TITLE
SMTP settings partially saved and functionnal

### ISSUE THREAD

**Issue by IwishIcanFLighT** [mriedmann/humhub-docker#43]
Hi,

When I try to set the SMTP settings for notifications, I do receive the "Saved" snackbar but the mail setting page resets itself to the defaults parameters:

![](https://puu.sh/ANXdA/a131203e90.png)

What's even weirder is that if I check the config file `dynamic.php`, my SMTP settings are here:

```
'mailer' =>
array (
  'transport' =>
  array (
    'class' => 'Swift_SmtpTransport',
    'host' => 'mail.domain.net',
    'username' => 'my_correct_username',
    'password' => 'my_correct_password',
    'encryption' => 'ssl',
    'port' => '465',
  ),
```

And trying to send an email such as password reset, it is indeed sent correctly but from the wrong name `humhub@example.com` and wrong address `social@example.com` which is not ideal.

Any help is appreciated.
Thanks

**Comment by mriedmann** [comment]
Sorry for the delay. I hope I will find some time today to look into this issue.

**Comment by mriedmann** [comment]
I am really sorry but I was unable to reproduce your problem. Could you provide some more information about your setup (versions, logs, compose-file, docker run arguments, etc.)?

A new version of this image was released moments ago. It would be a good idea to test this issue on the newest version (1.2.8), if possible.

**Comment by mriedmann** [comment]
A new version of humhub is out (1.3.x) and therefor a renewed version of this image will be released soon ([version-1.3](https://github.com/mriedmann/humhub-docker/tree/version-1.3)). I am closing this issue for now, please feel free to reopen it if the problem still exists with the new "experimental"-tag.
